### PR TITLE
Created the src folder with an init file and the df_prepper.py file c…

### PR DESCRIPTION
**Pull request by alex-is-learning**
…ontaining functions for taking the raw UN data and creating a flattened df

### CODE REVIEW

File: src/df_prepper.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import pandas as pd
+import numpy as np
+import os
```
Comment (Simon-Lee-UK) [inline src/df_prepper.py]: Does this get used anywhere in the module?

File: src/df_prepper.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	df : pandas dataframe
+		The first .csv file of the 10 to be combined
+	verbose : boolean
+		If True, print extra information for debugging
+ 	
+ 	Returns
+ 	-------
+ 	df : pandas dataframe
+ 		 The combined .csv files as a dataframe, replacing the input
+	'''
```
Comment (Simon-Lee-UK) [inline src/df_prepper.py:20]: Nice docstrings all the way through

**Review by Simon-Lee-UK — COMMENTED** [review]
If this works without importing os, remove that line then happy it can be merged into main.

**Review by alex-is-learning — COMMENTED** [review]
Reviewed Simon's comments

**Review by Simon-Lee-UK — APPROVED** [review]
Nice one, approved now if you want to merge

**Comment by alex-is-learning** [comment]
Merging pull request to add in the df_prepper file and function

**Pull request merged**